### DISCUSSION TITLE
Spaced repetition and DataView inline fields coexistence

### DISCUSSION THREAD

**Discussion by jtsao22** [st3v3nmw/obsidian-spaced-repetition#809] [Q&A] [answered]
Hello, first of all, thank you so much for this plugin. I use it everyday, and have gotten so much out of it.

So my use case is this:
I use spaced repetition and DataView's inline fields (https://blacksmithgu.github.io/obsidian-dataview/annotation/add-metadata/ search for inline fields).

The DataView inline fields are separated by "::", which is the same for spaced repetition cards (at least single line ones). 

So what's happening is that my Dataview inline fields are being included with my flashcards. Is there an easy way around this that you know of?

If not, is there anyway to limit the flashcards to a particular section of a note rather than parsing the entire note?
For instance, when writing book notes, I have a section for Chapter summaries where I use the Dataview inline notes, and then a separate section for Vocabulary where I use the flashcards. If I could just limit the flashcards to parse the Vocabulary section, my problem would be solved.

Thank you in advance!

**Comment by ronzulu** [comment, marked as answer]
Hey @jtsao22 

You can change the single line separator from the default "::" from OSR options, or would that cause you other issues?

![image](https://github.com/st3v3nmw/obsidian-spaced-repetition/assets/75528127/2674d0ee-364c-4b0f-869f-7e82ee004669)

**Reply by jtsao22** [reply]
That's perfect, thank you!
